### PR TITLE
docs: add setup validation howto to kube-proxy-free guide

### PR DESCRIPTION
**Pull request by borkmann**
See commit msg.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/10086)

### CODE REVIEW

**Comment by borkmann** [comment]
test-docs-please

**Comment by borkmann** [comment]
test-docs-please

**Comment by borkmann** [comment]
test-docs-please

**Comment by borkmann** [comment]
test-docs-please

File: Documentation/gettingstarted/kubeproxy-free.rst [hunk 1]
```diff
@@ -122,17 +122,137 @@ that we do not rely on a (non-existing) fallback.
...
+After deploying Cilium with above Quick-Start guide, we can first validate that
+the Cilium agent is running in the desired mode:
 
-In addition, thanks to the :ref:`host-services` feature, the NodePort service can
-be accessed by default from a host or a Pod within a cluster via it's public,
-cilium_host device or loopback address, e.g. ``127.0.0.1:NODE_PORT``.
+.. parsed-literal::
+
+    kubectl exec -it -n kube-system cilium-fmh8d -- cilium status | grep KubeProxyReplacement
+    KubeProxyReplacement:   Strict   [NodePort, ExternalIPs, HostReachableServicesTCP, HostReachableServicesUDP]
```
Comment (brb) [inline Documentation/gettingstarted/kubeproxy-free.rst]: Probably you can already apply the new output from: https://github.com/cilium/cilium/pull/10083.

File: Documentation/gettingstarted/kubeproxy-free.rst [hunk 2]
```diff
@@ -122,17 +122,137 @@ that we do not rely on a (non-existing) fallback.
...
+
+As a next, optional step, we deploy nginx pods, create a new NodePort service and
+validate that Cilium installed the service correctly.
+
+The following yaml is used for the backend pods:
+
+.. parsed-literal::
```
Comment (brb) [inline Documentation/gettingstarted/kubeproxy-free.rst:141]: We can save some space by replacing the yamls with the following cmds:
```
kubectl run nginx --generator=run-pod/v1 --image=nginx --replicas=2
kubectl expose deployment nginx --type=NodePort --port=80
```

**Review by brb — CHANGES_REQUESTED** [review]
LGTM, just two nits.

**Comment by borkmann** [comment]
test-docs-please

**Review by brb — APPROVED** [review]

**Pull request merged**